api/src/main/java/org/openmrs/api/context/Daemon.java

```diff
@@ -302,32 +369,32 @@ public void run() {
 	 */
 	public static void runInDaemonThreadAndWait(final Runnable runnable, DaemonToken token) {
 		Thread daemonThread = runInDaemonThread(runnable, token);
-		
+
 		try {
 			daemonThread.join();
 		}
 		catch (InterruptedException e) {
 			//Ignore
 		}
 	}
-	
```

Comment:
Did you intentionally do all the formatting changes in this class?

Reply:
No, I hope I have solved that in the next commit...